src/olympia/editors/forms.py

```diff
@@ -309,8 +309,8 @@ class ReviewForm(happyforms.Form):
                                         u'email)'))
     adminflag = forms.BooleanField(required=False,
                                    label=_(u'Clear Admin Review Flag'))
-    clear_info_request = forms.BooleanField(
-        required=False, label=_(u'Clear more info requested flag'))
+    info_request = forms.BooleanField(
+        required=False, label=_(u'Is more info requested?'))
```

Comment:
Is this the label shown to the reviewer? I think this might be slightly confusing, would "Request information from the developer" work?